ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -112,23 +110,22 @@ protected static void setupEthSigner(final long chainId) throws IOException, Cip
     httpServerOptions.setPort(serverSocket.getLocalPort());
     httpServerOptions.setHost("localhost");
 
-    final Web3j web3j =
-        new JsonRpc2_0Web3j(
-            new HttpService(
-                "http://"
-                    + httpClientOptions.getDefaultHost()
-                    + ":"
-                    + httpClientOptions.getDefaultPort()),
-            2000,
-            defaultExecutorService());
+    final HttpService web3jService =
+        new HttpService(
+            "http://"
+                + httpClientOptions.getDefaultHost()
+                + ":"
+                + httpClientOptions.getDefaultPort());
+    final Web3j web3j = new JsonRpc2_0Web3j(web3jService, 2000, defaultExecutorService());
+    final Eea eea = new JsonRpc2_0Eea(web3jService);
```

Comment:
They are different because the eea web3j rpc factory only exposes the one constructor which uses default values for the block time and executor service.